shared/src/commonMain/kotlin/io/customer/shared/util/LogLevel.kt

```diff
@@ -0,0 +1,29 @@
+package io.customer.shared.util
+
+/**
+ * Enum class to differentiate between log types and making it easier to filter.
+ */
+enum class LogLevel {
```

Comment:
Just trying to follow simpler naming convention and avoid CIO or CustomerIO in class naming as packages already help us filter sdk classes. But still open for better naming conventions.